sunpy/time/utime.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import absolute_import
+
+from astropy.time.formats import erfa, TimeFromEpoch
+
+
+class TimeUTime(TimeFromEpoch):
+    """Seconds from 1979-01-01 00:00:00 UTC. Same as Unix time
```

Comment:
I forget, should there not be a new line after the docstring quotes?

Reply:
[PEP257](https://www.python.org/dev/peps/pep-0257/#id17) doesn't specify but I can change it if that's a standard recommended in sunpy or in the science stack.

Reply:
It's not an issue then!

Reply:
It's a Cadair standard, which means most of SunPy is like that ;)